packages/flutter/test/material/reorderable_list_test.dart

```diff
@@ -1601,6 +1601,24 @@ void main() {
       );
```

Comment:
Can you check when the list is reversed that these test scenarios still work where relevant?